snakesee/parser/core.py

```
@@ -274,15 +274,15 @@ def parse_rules_from_log(log_path: Path) -> dict[str, int]:
     return rule_counts
 
 
-def parse_running_jobs_from_log(
+def parse_running_jobs_from_log(  # noqa: C901
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `C901`)

Remove unused `noqa` directive

(RUF100)